Makefile

```diff
@@ -21,11 +21,11 @@ pytest: build
 	cd package && python3 setup.py test
 
 e2e-tests: build
-	cd package && behave
+	cd package && behave --tags="not @wip"
 
 pylint:
 	cd package && isort
-	pylint -j 0 --disable=unnecessary-pass package/kedro_viz
+	pylint -j 0 --disable=bad-continuation,unnecessary-pass package/kedro_viz
```

Comment:
why

Reply:
Linter started complaining `wait_for` as soon as I moved from `/features/steps` to `utils.py`. It was asking me to add 4 spaces to the argument of `wait_for`, which was not quite correct (I double-checked with space/tab etc. ). Maybe it is a problem of `.pylintrc` not being used? I am not quite sure why...